Partition (WBP)/Files/VTLI0036C.LANSA.file.yml

```diff
@@ -0,0 +1,289 @@
+﻿File:
+  Name:                           VTLI0036C
+  File Type:                      PC Other File
+  Library:                        LANSA
+  Record Format Name:             VTLI0036C
+  Module Library:                 Same Library As File
+  File Uses SQL on IBM i:         No
+  Enabled For RDMLX:              Yes
+  Enabled For Long Names:         Yes
+  Share Data Path:                No
+  Secure From File Overrides:     Yes
+  Strip Debug On Compile:         Yes
+  Suppress IOM0034 Message:       No
+  Ignore Decimal Data Error:      No
+  Create I/O Module:              Yes
+  Create Batch Control:           No
+  IBM i High Speed Table:         No
+  Auto RRN Generation:            No
+  Create RRNO Column:             No
+  Auto RRNO Is Fixed:             No
+  Convert Special Characters:     Yes
+  Enable Commitment Control:      No
+  Auto Commit:                    No
+  Parameters:                     SIZE( 10000 2000 3) LVLCHK(*YES)
+  Read-only Access:               No
+  Enable Database Triggers:       No
+
+  Multilingual Details:
+  - DAN:
+      Description:        "VTLI0036C"
```

Comment:
ODBCORACLE?

Reply:
Descriptions I am manually  editing the file by opening in Notepad, is that fine ?

Reply:
Yes it is provided its validly displayed in the IDE